include/zephyr/net/net_context.h

```diff
@@ -370,6 +370,10 @@ __net_socket struct net_context {
 		/** Receive network packet information in recvmsg() call */
```

Comment:
Commit message of `net: context: Add support for setting receive hoplimit option` should mention that this also supports IPv4 `IP_RECVTTL` option.